src/gui/properties/trackerlistwidget.cpp

```diff
@@ -683,3 +684,19 @@ void TrackerListWidget::displayColumnHeaderMenu()
 
     menu->popup(QCursor::pos());
 }
+
+void TrackerListWidget::wheelEvent(QWheelEvent *event)
+{
+    if (event->modifiers() & Qt::ShiftModifier)
+    {
+        // Shift + scroll = horizontal scroll
+        event->accept();
+        QWheelEvent scrollHEvent(event->position(), event->globalPosition()
+            , event->pixelDelta(), event->angleDelta().transposed(), event->buttons()
+            , event->modifiers(), event->phase(), event->inverted(), event->source());
```

Comment:
```suggestion
        QWheelEvent scrollHEvent {event->position(), event->globalPosition()
            , event->pixelDelta(), event->angleDelta().transposed(), event->buttons()
            , event->modifiers(), event->phase(), event->inverted(), event->source()};
```

Reply:
The original instances still follow scrollHEvent(), should they be updated as well?

Reply:
> The original instances still follow scrollHEvent(), should they be updated as well?

Well, you don't have to do it, but you can do it if you want to.

Reply:
Done.